src/ParaTest/Runners/PHPUnit/BaseRunner.php

```diff
@@ -72,7 +79,12 @@ public function run()
         $this->verifyConfiguration();
         $this->initCoverage();
         $this->load();
-        $this->printer->start($this->options);
+
+        // Print the starting messages only if they were not printed yet.
+        if (!$this->starting_messages_printed) {
```

Comment:
you don't need that comment if you just extract that code to a new method. The next dev would then read
`$this->printStartingMessageIfNotPrintedYet();`
... that's at least stable for changes and refactorings